server/lib/awsClient.js

```diff
@@ -0,0 +1,17 @@
+import config from 'config';
+import aws from 'aws-sdk';
+
+// S3 bucket
+let awsClient;
+
+if (config.aws.s3.key) {
+  awsClient = aws.config.update({
```

Comment:
Looks like `aws.config.update` is not returning anything, leading to `awsClient` being undefined.